### DISCUSSION TITLE
Apipie

### DISCUSSION THREAD

**Discussion by bogdan-boer** [Wolfpack-Digital/rails-template#25] [Ideas]
Hello, 

Since we are using apipie for documentation for most projects maybe it would be a good idea to do the initial setup in this repo so it won't be done manually for each project setup, what do you think?

**Comment by siklodi-mariusz** [comment]
Well, this is strange. Until now I was under the impression that Apipie is being set up by the template but it seems it's not. Although it assumes there is Apipie installed since it adds these spec support files https://github.com/Wolfpack-Digital/rails-template/blob/0ab7d3eea127dae76c6db4729504d4e9de8aa5b4/spec/support/apipie_helper.rb

So yeah, I think it should be added  👍

**Reply by siklodi-mariusz** [reply]
Opened Issue #26, can you @bogdan-boer take care of this?

**Reply by bogdan-boer** [reply]
Yep I will
